### PR TITLE
Ported Web Services tab to Redux-based implementation

### PR DESCRIPTION
**Pull request by philliphoff**
This is a straight port of the existing Web Services tab (pure JSX) to a Redux-based implementation.  The only alteration was substitution of its master/detail "tri panel" with the generic one used by the Data tab.

Note: no unit tests were created for this PR, but they will come along on subsequent PRs which actually add new features.

### CODE REVIEW

File: src/request/components/RequestDetailPanelWebServices.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                <tbody>
+                    {requestItems}
+                </tbody>
+            </table>
+        );
+    }
+}
+
+/**
+ * React class to display service details
```
Comment (mike-kaufman) [inline src/request/components/RequestDetailPanelWebServices.tsx]: service details?

Comment (mike-kaufman) [inline src/request/components/RequestDetailPanelWebServices.tsx]: isn't this "service headers" or "web service headers"?  (I guess I'm confused on where the "details" bit of the comes from)

**Comment by mike-kaufman** [comment]
lgtm, but I haven't been following the client UI stuff closely so @avanderhoorn should sign off.

File: src/request/components/RequestDetailPanelWebServices.tsx (hunk 1 above)
Comment (philliphoff) [inline src/request/components/RequestDetailPanelWebServices.tsx]: I didn't do any renaming of the components; at the moment those panels show only the request/response headers but more "details" will eventually shown (e.g. body, cookies, etc.).

File: src/request/request-tab.js [hunk 2]
```diff
@@ -35,9 +35,16 @@ module.exports = {
 
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
-require('./components/request-detail-panel-webservices');
```
Comment (avanderhoorn) [inline src/request/request-tab.js]: Can this file be removed now?

Comment (philliphoff) [inline src/request/request-tab.js]: Yep, it *was* but was brought back to life due to a merge conflict.  It should be dead again.

**Pull request merged**